brodata/gld.py

```diff
@@ -70,15 +70,22 @@ def get_objects_as_csv(
     and fetches the data in CSV format. The `rapportagetype` and `observatietype`
     parameters can be used to filter the data.
     """
-    url = f"{GroundwaterLevelDossier._rest_url}/objectsAsCsv/{bro_id}"
-    params = {
-        "rapportagetype": rapportagetype,
-    }
-    if observatietype is not None:
-        params["observatietype"] = observatietype
-    req = requests.get(url, params=params)
+    if bro_id.startswith("http"):
```

Comment:
so bro_id can be both an id and a url? And in case it is a URL the rapportagetype and observatietype are not added to the request. Is this undocumented on purpose?

Reply:
Yes, now bro_id can be a url as well. I will add this to the docstring.

The gm-webservice generates the url, with filled in params, so we cannot (and do not want to) add them to the request.